_config.yml

```diff
@@ -1005,6 +1009,12 @@ vendors:
   fancybox:
   fancybox_css:
 
+  # Internal version: 1.0.4
+  # See: https://github.com/francoischalifour/medium-zoom
+  # Example:
+  # mediumzoom: //cdn.jsdelivr.net/npm/medium-zoom@1/dist/medium-zoom.min.js
```

Comment:
I think the example link should be 1.0.4 since internal version is 1.0.4.
https://cdn.jsdelivr.net/npm/medium-zoom@1.0.4/dist/medium-zoom.min.js

Reply:
https://github.com/theme-next/hexo-theme-next/pull/543

https://github.com/theme-next/hexo-theme-next/pull/543#discussion_r244306787